scipy/optimize/tests/test_optimize.py

```diff
@@ -2308,3 +2309,31 @@ def fquad_jac(x):
             hess = fquad_hess
         res = optimize.minimize(fquad, x0, method=meth, jac=jac, hess=hess)
         assert_allclose(res.x, np.arange(np.size(x0)), atol=2e-4)
+
+
+@pytest.mark.parametrize('method', ['Powell', 'L-BFGS-B', 'SLSQP',
+                                    'trust-constr'])
```

Comment:
I think we'll need more comprehensive tests, as this currently passes in master for Powell and trust-constr.

Reply:
I can't remember from the original issue if Powell and trust-constr were sensitive to this. I can only remember that lbfgsb and slsqp were.